### ISSUE TITLE
How to prevent right click ?

### ISSUE THREAD

**Issue by adham-ashraf77** [jnschulze/flutter-webview-windows#266]
anyone help to prevent right-click in the webview I show YouTube video when right click can copy the link to video  ?

**Comment by adham-ashraf77** [comment]
![image](https://github.com/jnschulze/flutter-webview-windows/assets/55947445/dfa1ca64-7717-4598-9d72-47f8e80d0a78)
this is what I mean want to prevent showing the menu

**Comment by vamsikrishna-vk** [comment]
Only right click or don't want to allow interactions?

**Comment by adham-ashraf77** [comment]
> Only right click or don't want to allow interactions?

Yes please don't want this menu because it gets URL video from it 
@vamsikrishna-vk

**Comment by adham-ashraf77** [comment]
is there any news @vamsikrishna-vk
